lib/rexml/entity.rb

```diff
@@ -59,6 +60,10 @@ def initialize stream, value=nil, parent=nil, reference=false
         @name = stream
         @value = value
       end
+
+      if PEREFERENCE_RE.match?(@value)
+        raise "Parameter Entity References forbidden in internal subset: #{@value}"
```

Comment:
* Could you use `ArgumentError` instead of `RuntimeError`?
* Could you avoid needless capital case?

```suggestion
        raise ArgumentError, "Parameter entity references forbidden in internal subset: #{@value}"
```

BTW, do we need a validation here? This method document says the following:

> +WARNING+: There is no validation of entity state except when the entity
> is read from a stream.  If you start poking around with the accessors,
> you can easily create a non-conformant Entity.

Reply:
@kou 
OK.
I see.
We prefer not to add this validation in `.new`.

Similarly, I don't think this validation should be added to `#write`.
What do you think?

Reply:
I agree with you.